docs/docs/data-structures/tree/schema-evolution/index.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: Schema Evolution
+sidebar_position: 8
+---
+
+# Schema Evolution
+
+As your application grows and evolves, you may need to modify the structure of your [SharedTree](../index.mdx) data.
+SharedTree makes it possible to update your [schemas](../schema-definition.mdx) in ways that allow an app to load or upgrade documents created or edited by earlier versions of the app (see [understanding compatibility](./index.mdx#understanding-compatibility)).
+
+> **Note:** For detailed information about which types of changes are safe versus breaking, see [types of schema changes](./types-of-changes).
```

Comment:
This seems like a good use case for [admonition](https://docusaurus.io/docs/markdown-features/admonitions) syntax :)

Reply:
Same for the new note in `schema-definition`.